gfx.cpp

```diff
@@ -1963,7 +1975,7 @@ class GfxInternal
         width = GFX_MAX(width, 1u);
         height = GFX_MAX(height, 1u);
         mip_levels = GFX_MIN(GFX_MAX(mip_levels, 1u), gfxCalculateMipCount(width, height));
-        D3D12_RESOURCE_STATES const resource_state = D3D12_RESOURCE_STATE_COPY_DEST;
+        D3D12_RESOURCE_STATES const resource_state = D3D12_RESOURCE_STATE_COMMON;
```

Comment:
All resources are created in COMMON state. The previous behaviour was actually a bug as GPU buffers where generally used as the dest for a copy operation when loading data into them which was previously implicitly setting the state to COPY_DEST. However if this operations was not done then this initial state would be incorrect